src/config.c

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright Redis Ltd. 2016 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
+#include <strings.h>
+#include "config.h"
+
+// Default configuration values.
+// All values provided by CONFIG PRD
+// (https://redislabs.atlassian.net/wiki/spaces/DX/pages/3980198010/PRD+CONFIG+for+modules+arguments)
```

Comment:
This, probably, shouldn't be here :-)
I understand this would help, but not everyone in the world...